about.html

```diff
@@ -71,7 +71,278 @@
+                появившимися на рынке.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="main__wr-about-company-info">
+          <div class="main__container">
+            <div class="main__about-company-info">
+              <div class="main__company-info-title">
+                <h2 class="main__company-title">О компании</h2>
+                <img src="./assets/images/about/crystal.png" alt="crystal" />
+              </div>
+              <div class="main__wr-company-descr">
+                <p class="main__company-descr">
+                  Мinerals Operating – это молодая компания, которая оказывает
+                  услуги в сфере недропользования на территории Казахстана. Мы
+                  одна из первых компаний являющихся лицензированным
+                  консультантом в сфере минеральных ресурсов (лицензия
+                  №AFSA-A-LA-2020-0004). Мinerals Operating зарегистрирована в
+                  Международном Финансовом Центре «Астана» (МФЦА) и осуществляет
+                  деятельность, основанную на нормах и принципах Английского
+                  общего права. МФЦА – это уникальный центр, который использует
+                  лучшие практики и подходы международных финансовых центров и
+                  сотрудничает с крупными корпорациями и небольшими компаниями,
+                  недавно появившимися на рынке. Органы управления МФЦА
+                  предлагают предпринимателям возможности для привлечения
+                  инвестиций и гарантии для поддержки бизнеса, выгодный
+                  налоговый режим и льготы для бизнес-операций.
+                </p>
+                <p class="main__company-descr">
+                  Современное оборудование для разработки месторождений, штат
+                  опытных сотрудников, помощь в подготовке проектной
+                  документации в соответствии с нормами законодательства – вот
+                  то, на что вы можете рассчитывать. Мы работаем в сфере
+                  Оборудование для разработки месторождений, штат опытных
+                  сотрудников, помощь в подготовке проектной документации в
+                  соответствии с нормами законодательства – вот то, на что вы
+                  можете рассчитывать. Органы управления МФЦА предлагают
+                  предпринимателям возможности для привлечения инвестиций и
+                  гарантии для поддержки бизнеса, выгодный налоговый режим и
+                  льготы для бизнес-операций. Органы управления МФЦА предлагают
+                  предпринимателям возможности для привлечения
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="main__company-principles">
+            <div class="main__container">
+              <h2 class="main__principles-title">Принципы компании</h2>
+              <div class="main__wr-principles-box">
+                <div class="main__principles-box">
```

Comment:
Почему бы не сделать это блоком?
```suggestion
                <div class="principles-box">
```